styles.css

```diff
@@ -1178,3 +1175,328 @@ button {
 #msg:hover {
   border: 2px solid blue;
 }
+
+.bottom-nav {
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  width: 327px;
+  height: 72px;
+  justify-content: center;
+  padding: 10px;
+  left: 10px;
+  top: 6290px;
+}
+
+@media only screen and (min-width: 768px) {
+  .toolbar {
+    width: 1280px;
+    height: 3250px;
+    margin-left: 30px;
+  }
```

Comment:
- [ ] The width of the content of your website is less than the width of the viewport.  Please make sure the width of every element in the website doesn't exceed the size of the viewport. 

Your Design 👇 
![image](https://user-images.githubusercontent.com/63473584/178084005-cb4af92d-9977-47ae-8cb5-8c83aff46291.png)


The issue is as a result of the fixed-width `width: 1280px;` you added. I would recommend you use percentage (%) or VW to dynamically adjust your page to match the viewport. 

Example:  
```
.toolbar {
    width: 100%
  }
```